src/leap/mail/tests/test_mail.py

```diff
@@ -0,0 +1,378 @@
+# -*- coding: utf-8 -*-
```

Comment:
Many tests here fail, but I guess we can merge the pull-req and take care of the failing tests afterwards.